packages/bpk-component-calendar/src/BpkCalendarNav.js

```diff
@@ -81,17 +84,20 @@ const BpkCalendarNav = props => {
         >
           <ArrowLeftIcon className={getClassName('bpk-calendar-nav__icon')} />
           <span className={getClassName('bpk-calendar-nav__text--hidden')}>
-            {formatMonth(addMonths(baseMonth, -1))}
+            {previousMonthLabel || formatMonth(prevMonth)}
           </span>
         </button>
       </div>
       <div className={getClassName('bpk-calendar-nav__month')}>
         <label
           htmlFor={`${id}_select`}
           className={getClassName('bpk-calendar-nav__text--hidden')}
+          aria-hidden="true"
```

Comment:
The label is repeated again in the BpkSelect, by using `aria-hidden="true"` it's still used in the select, but is no longer read twice.